wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -0,0 +1,791 @@
+
+*Not to be confused with [Notice](#notice).*
+
+Hatnotes are short notes placed at the top of an article or section to help readers navigate to related articles or inform them about related topics.
+
+Hatnotes must be italicised and be placed immediately after the heading. If multiple hatnotes are used, they must be on the same paragraph separated with a line break.
+
+### Main page
+
+*Main page* hatnotes direct a reader to the main article of the topic. When this hatnote is used, it implies that the section it is on is a summary of what the linked page is about. This hatnote should have only one link, but may use more if needed. These must be formatted as follows:
+
+```markdown
+*Main page: {article}*
+
+*Main pages: {article} and {article}*
+```
+
+### See also
+
+*See also* hatnotes suggest to readers other points of interest from a given article or section. These must be formatted as follows:
+
+```markdown
+*See also: {article}*
+
+*See also: {article} and {article}*
+```
+
+### For see
+
+*For see* hatnotes are similar to *see also* hatnotes, but are generally more descriptive and direct. This hatnote should have only one link, but may use more if needed. These must be formatted as follows:
+
+```markdown
+*For {description}, see: {article}`*
+
+*For {description}, see: {article} and {article}`*
+```
+
+### Not to be confused with
+
+*Not to be confused with* hatnotes help distinguish ambiguous or misunderstood article titles or sections. This hatnote should have only one link, but may use more if needed. These must be formatted as follows:
+
+```markdown
+*Not to be confused with {article}.*
+
+*Not to be confused with {article} or {article}.*
+```
+
+### For other uses
+
+*For other uses* hatnotes are similar to *not to be confused with* hatnotes, but links directly to the disambiguation article. This hatnote must only link to the disambiguation article. These must be formatted as follows:
+
+```markdown
+*For other uses, see {disambiguation article}.*
+```
+
+## Notice
+
+*Not to be confused with [Hatnote](#hatnote).*
+
+A notice should be placed where appropriate in a section, but must start off the paragraph and use italics. Notices may contain bolding where appropriate, but should be kept to a minimum. Notices must be written as complete sentences. Thus, unlike most [hatnotes](#hatnotes), must use a full stop (`.`) or an exclamation mark (`!`) if appropriate. Anything within the same paragraph of a notice must also be italicised. These must be formatted as follows:
+
+```markdown
+*Note: {note}.*
+
+*Notice: {notice}.*
+
+*Caution: {caution}.*
+
+*Warning: {warning}.*
+```
+
+- `Note` should be used for factual or trivial details.
+- `Notice` should be used for reminders or to draw attention to something that the reader should be made aware of.
+- `Caution` should be used to warn the reader to avoid unintended consequences.
+- `Warning` should be used to warn the reader that action may be taken against them.
+
+### Notice to
```

Comment:
I think it's very, very specific and should be deprecated in favor of general notes/notices/whatever, since the target audience is already reading the article (and these who find it irrelevant may just skip it like any other note)